dist/profile/templates/confluence/vhost.conf

```diff
@@ -2853,12 +2853,86 @@ RewriteRule "^/display/JENKINS/Subversion\+Plugin$" "https://plugins.jenkins.io/
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+# Perforce plugin no longer exists apparently, so redirect to p4
+RewriteRule "^/display/JENKINS/Perforce\+Plugin$" "https://plugins.jenkins.io/p4" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Xcode\+Plugin$" "https://plugins.jenkins.io/xcode-plugin" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Embeddable\+Build\+Status\+Plugin$" "https://plugins.jenkins.io/embeddable-build-status" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/SSH\+Credentials\+Plugin$" "https://plugins.jenkins.io/ssh-credentials" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Pipeline\+Groovy\+Plugin$" "https://plugins.jenkins.io/workflow-cps" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Git\+Changelog\+Plugin$" "https://plugins.jenkins.io/git-changelog" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Robot\+Framework\+Plugin$" "https://plugins.jenkins.io/robot" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/SSH\+Slaves\+plugin$" "https://plugins.jenkins.io/ssh-slaves" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Bitbucket\+Push\+And\+Pull\+Request\+Plugin$" "https://plugins.jenkins.io/bitbucket-push-and-pull-request" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Checkstyle\+Plugin$" "https://plugins.jenkins.io/checkstyle" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Script\+Security\+Plugin$" "https://plugins.jenkins.io/script-security" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Git\+Client\+Plugin$" "https://plugins.jenkins.io/git-client" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Gradle\+Plugin$" "https://plugins.jenkins.io/gradle" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Bitbucket\+Branch\+Source\+Plugin$" "https://plugins.jenkins.io/cloudbees-bitbucket-branch-source" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Docker\+Pipeline\+Plugin$" "https://plugins.jenkins.io/docker-workflow" [NC,L,QSA,R=301]
+# RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+# RewriteRule "^/display/JENKINS/Dynamic\+Parameter\+Plug-in$" "" [NC,L,QSA,R=301]
```

Comment:
what's going on here?

Reply:
Plugin distribution was suspended due to security vulnerabilities.  No need to redirect anywhere?

Reply:
I started to look up the link and not find a target. I'll just remove it.